hypothesis-python/tests/nocover/test_floating.py

```diff
@@ -24,7 +24,11 @@
 from tests.common.utils import fails
 
 TRY_HARDER = settings(
-    max_examples=1000, suppress_health_check=[HealthCheck.filter_too_much]
+    max_examples=1000,
+    suppress_health_check=[HealthCheck.filter_too_much],
+    # Many of these tests expect to fail, so they need to be derandomized to
+    # prevent occasional flakiness.
+    derandomize=True,
```

Comment:
This test file has the same flakiness problem as the long binary strings, but there are enough tests here that I don't want to convert them all into `quality` tests right now, so instead I'm just bailing and setting `derandomize=True`.

Reply:
I thought this was a flakiness issue, but it seems that two tests are consistently failing on Python 3.8 on all platforms:
```
FAILED tests/nocover/test_floating.py::test_can_generate_really_small_positive_floats
FAILED tests/nocover/test_floating.py::test_can_generate_really_small_negative_floats
```